tests/test-bundler.js

```diff
@@ -8,33 +8,45 @@ import chaiEnzyme from 'chai-enzyme'
 
 // Mocha / Chai
 // ------------------------------------
-mocha.setup({ ui: 'bdd' })
-chai.should()
+mocha.setup({ ui: 'bdd' });
+chai.should();
 
-global.chai = chai
-global.expect = chai.expect
-global.sinon = sinon
+global.chai = chai;
+global.expect = chai.expect;
+global.sinon = sinon;
 
 // Chai Plugins
 // ------------------------------------
-chai.use(chaiEnzyme())
-chai.use(dirtyChai)
-chai.use(chaiAsPromised)
-chai.use(sinonChai)
+chai.use(chaiEnzyme());
+chai.use(dirtyChai);
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
 
 // Test Importer
 // ------------------------------------
 // We use a Webpack global here as it is replaced with a string during compile.
 // Using a regular JS variable is not statically analyzable so webpack will throw warnings.
-const testsContext = require.context('./', true, /\.(spec|test)\.(js|ts|tsx)$/)
+const serverSideTestsContext = require.context('./', true, /\.(spec|test)\.(js|jsx)$/);
```

Comment:
Now have two sources of tests (server side tests in 'tests' folder and client side tests each near it's component.